olive/systems/azureml/aml_system.py

```diff
@@ -33,17 +33,17 @@ class AzureMLSystem(OliveSystem):
 
     def __init__(
         self,
-        aml_config_path: str,
+        azureml_client_config: AzureMLClientConfig,
```

Comment:
Should we also keep the legacy `aml_config_path` before the releasement?

Reply:
Even we keep this, aml system wont't work with aml_config_path as we changed the mlclient creation logic inside. Do we want to keep this?